A62-pick-first.md

```diff
@@ -9,63 +9,68 @@ A62: `pick_first` LB policy
 
 ## Abstract
 
-This design specifies a configuration option for the `pick_first` LB policy that
-would enable it to randomly shuffle the order of the addresses it receives
-before it starts to attempt to connect.
-
-Given that `pick_first` is the default LB policy in all gRPC implementations, it
-is necessary to have a written specification that details its behavior. This
-document provides that specification.
+This document describes a couple of changes being made to the `pick_first` LB
+policy with regards to
+- Expected behavior when connections to all addresses fail, and
+- Support for address order randomization
 
 ## Background
```

Comment:
The gRFC template has a [Related Proposals](https://github.com/grpc/proposal/blob/master/GRFC-TEMPLATE.md#related-proposals) subsection underneath the "Background" section.  Please include that here, and list gRFCs A37 and A56.